HISTORY.rst

```diff
@@ -0,0 +1,28 @@
+.. :changelog:
```

Comment:
I would remove this file and let [towncrier](https://pypi.org/project/towncrier/) to the job w/ commit msgs